docs/user/exercises/programming-exercise-setup.rst

```diff
@@ -187,28 +187,201 @@ Exercise Creation
+|                 | metrics or size limitations            +--------------------------+-----------------------+
+|                 |                                        | Checkstyle sizes         |                       |
++-----------------+----------------------------------------+--------------------------+-----------------------+
+| Documentation   | Code with missing or flawed            | Checkstyle javadoc       |                       |
+|                 | documentation                          +--------------------------+-----------------------+
+|                 |                                        | Checkstyle annotation    |                       |
+|                 |                                        +--------------------------+-----------------------+
+|                 |                                        | PMD Documentation        |                       |
++-----------------+----------------------------------------+--------------------------+-----------------------+
+| Naming & Format | Rules that ensure the readability      | Checkstyle imports       |                       |
+|                 | of the source code (name conventions,  +--------------------------+-----------------------+
+|                 | imports, indentation, annotations,     | Checkstyle indentation   |                       |
+|                 | white spaces)                          +--------------------------+-----------------------+
+|                 |                                        | Checkstyle naming        |                       |
+|                 |                                        +--------------------------+-----------------------+
+|                 |                                        | Checkstyle whitespace    |                       |
++-----------------+----------------------------------------+--------------------------+-----------------------+
+| Miscellaneous   | Uncategorized rules                    | Checkstyle miscellaneous |                       |
++-----------------+----------------------------------------+--------------------------+-----------------------+
+
+  .. note::
+    For Swift, only the category Code Style can contain code quality issues currently. All other categories displayed on the grading page are dummies.
+
+  - On the left side of the page, instructors can configure the static code analysis categories.
+
+    - **Category**: The name of category defined by Artemis
+    - **State**:
+      - ``INACTIVE``: Code quality issues of an inactive category are not shown to students and do not influence the score calculation
+      - ``FEEDBACK``: Code quality issues of a feedback category are shown to students but do not influence the score calculation
+      - ``GRADED``: Code quality issues of a graded category are shown to students and deduct points according to the Penalty and Max Penalty configuration
```

Comment:
This doesn't appear to work as intended:
![grafik](https://user-images.githubusercontent.com/72132281/107523785-643e7700-6bb5-11eb-8891-04f12f087f70.png)